homeassistant/components/mqtt/__init__.py

```diff
@@ -230,15 +230,15 @@ def publish_template(hass: HomeAssistantType, topic, payload_template,
 
 @asyncio.coroutine
```

Comment:
You need to remove this decorator.

Reply:
Oh noes :( sorry that I didn't catch that, it's been a long week...